.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] The README is the most important part of our repository, and we should always have a good looking and professional one. It should contain useful information, a description and tell the reader how to use the code and who are/is the maintainer(s). 

We should ensure always that every Repository we crate has a professional README, Let's use Microverse template to achieve that.

Reply:
:heavy_check_mark:  A professional readme has been added